src/CommunityToolkit.Maui.MediaElement/AppBuilderExtensions.shared.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CommunityToolkit.Maui.MediaElement;
+
+/// <summary>
+/// <see cref="MauiAppBuilder"/> Extensions
+/// </summary>
+public static class AppBuilderExtensions
+{
+	/// <summary>
+	/// Initializes the .NET MAUI Community Toolkit Core Library
+	/// </summary>
+	/// <param name="builder"><see cref="MauiAppBuilder"/> generated by <see cref="MauiApp"/> </param>
+	/// <returns><see cref="MauiAppBuilder"/> initialized for <see cref="CommunityToolkit.Maui.MediaElement"/>.</returns>
+	public static MauiAppBuilder UseMauiCommunityToolkitMediaElement(this MauiAppBuilder builder)
+	{
+		builder.ConfigureMauiHandlers(h =>
+		{
+			h.AddHandler<MediaElement, MediaElementHandler>();
+		});
+
+		return builder;
```

Comment:
Do we need to call Core library initializer here?

Reply:
I don't think so...? It doesn't even have a dependency on Core at this point.